### ISSUE TITLE
[help]after click images=>localhost:4200:get/undefined

### ISSUE THREAD

**Issue by budineverend** [MurhafSousli/ng-gallery#27]
![capture](https://user-images.githubusercontent.com/30884438/29151561-40f0283e-7dac-11e7-99c0-1c7ba5142e0f.PNG)

hallo,i cant resolved this problem is there someone can help me how to solve this is problem?
please...

thank you

**Comment by budineverend** [comment]
up

**Comment by MurhafSousli** [comment]
Hi @budineverend, please paste your code

**Comment by PsychicCat** [comment]
my code: 

``` 
ngOnInit() {
        this.photoSub = this._activatedRoute.data
            .flatMap((data) => {

                this.gallery.load([{
                    src: `image path`
                }]);

    }

    showModal(): void {
        this.gallery.set(0);
    }
```

**Comment by MurhafSousli** [comment]
This is probably fixed in v1.0.0! if you still have the issue, please reopen. closing for now..
